qiita_db/analysis.py

```diff
@@ -79,6 +79,26 @@ def _status_setter_checks(self, conn_handler):
             raise QiitaDBStatusError("Can't set status away from public!")
 
     @classmethod
+    def get_user_default(cls, user):
```

Comment:
Uhm... from my point of view, is usually better to ask the user: give me your default cart. It will be a instance method rather than a classmethod, which is already a signal of better design...